### PR TITLE
PAY-697 - CNP config for payment app.

### PR DESCRIPTION
**Pull request by smathangi**

### CODE REVIEW

File: infrastructure/prod.tfvars [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+spring_mail_host = "mta.reform.hmcts.net"
+spring_mail_port = "25"
+spring_mail_properties_mail_smtp_starttls_enable = "true"
+spring_mail_properties_email_smtp_ssl_trust = "*"
+card_payments_email_from = "no-reply@reform.hmcts.net"
+card_payments_email_to = "MiddleOffice.Developments@liberata.gse.gov.uk,Oluwakemi.Benson@hmcts.net,Leonie.Watson@Justice.gov.uk,kevin.harper@hmcts.net,ajaipal.chana@hmcts.net,phillip.probert@hmcts.net"
+card_payments_email_subject = "Card Payments Reconciliation Report"
+card_payments_email_message = "Hi <br/><br/>Please find attached today''s reconciliation report. <br/><br/>Regards <br/><br/>Payments team<br/><br/>"
+pba_payments_email_from = "no-reply@reform.hmcts.net"
+pba_payments_email_to = "moneyclaims@hmcts.gsi.gov.uk,Oluwakemi.Benson@HMCTS.NET,Leonie.Watson@Justice.gov.uk,kevin.harper@hmcts.net,ajaipal.chana@hmcts.net,phillip.probert@hmcts.net"
```
Comment (jaleen) [inline infrastructure/prod.tfvars]: can't do  that!

File: infrastructure/prod.tfvars [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+card_payments_report_schedule = "0 0 2 * * ?"
+card_payments_report_scheduler_enabled = "true"
+pba_payments_report_schedule = "0 0 2 * * ?"
+pba_payments_report_scheduler_enabled = "true"
+spring_mail_host = "mta.reform.hmcts.net"
+spring_mail_port = "25"
+spring_mail_properties_mail_smtp_starttls_enable = "true"
+spring_mail_properties_email_smtp_ssl_trust = "*"
+card_payments_email_from = "no-reply@reform.hmcts.net"
+card_payments_email_to = "MiddleOffice.Developments@liberata.gse.gov.uk,Oluwakemi.Benson@hmcts.net,Leonie.Watson@Justice.gov.uk,kevin.harper@hmcts.net,ajaipal.chana@hmcts.net,phillip.probert@hmcts.net"
```
Comment (jaleen) [inline infrastructure/prod.tfvars]: can't do that

File: infrastructure/variables.tf [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  default = "0 */30 * * * ?"
+}
+variable "pba_payments_report_scheduler_enabled" {
+  default = "true"
+}
+variable "pba_payments_email_from" {
+  default = "no-reply@reform.hmcts.net"
+}
+variable "pba_payments_email_to" {
+  default = "Oluwakemi.Benson@HMCTS.NET,Krishna.Nuthalapati@HMCTS.NET"
```
Comment (jaleen) [inline infrastructure/variables.tf]: can't add these here

File: infrastructure/prod.tfvars (hunk 1 above)
Comment (smathangi) [reply infrastructure/prod.tfvars]: true. it's a internal data - will add as vault secret.

**Pull request merged**